module.txt

```diff
@@ -1,9 +1,9 @@
 {
     "id" : "MetalRenegades",
     "version" : "0.1.0-SNAPSHOT",
-    "author" : "",
+    "author" : "Wabadump & AndyTechGuy",
     "displayName" : "MetalRenegades",
-    "description" : "",
+    "description" : "A harsh western world with a robotic twist. This is currently a work-in-progress GSOC project for 2019.",
```

Comment:
Change the second sentence to "A GSOC project of 2019" or similar, so in case you forget to get back to this later, you don't end up with invalid info.

Reply:
you don't need to state that you are working on it.

Reply:
Changed the second sentence to "A student project for GSOC 2019".